bindings/node.js/lib/kzg.js

```diff
@@ -23,7 +23,7 @@ function transformTrustedSetupJson(filePath) {
   const setupText =
     bindings.FIELD_ELEMENTS_PER_BLOB +
     "\n65\n" +
-    trustedSetup.setup_G1.map((p) => p.substring(2)).join("\n") +
+    trustedSetup.setup_G1_lagrange.map((p) => p.substring(2)).join("\n") +
```

Comment:
Kind of surprised this one line change is sufficient for the nodejs bindings. What happens to `setup_G1` now?

Reply:
The nodejs bindings convert the trusted setup from the specs to the c-kzg format. The `setup_G1` field will be unused now, like `setup_G1_lagrange` was and `roots_of_unity` are. I don't think we should modify that file.